docs/datascience/nb-in-the-browser.md

```diff
@@ -0,0 +1,66 @@
+---
+Order: 8
+Area: datascience
+TOCTitle: Jupyter Notebooks on VS Code browser
+ContentId: 0faf5b06-ddad-4594-8d5e-fa409c7da82c
+PageTitle: Jupyter Notebooks on VS Code browser
+DateApproved: 1/9/2023
+MetaDescription: Ways to work with Jupyter notebooks in the browser on VS Code.
+MetaSocialImage: images/tutorial/social.png
+---
+
+# Jupyter notebooks in VS Code running in the browser
+
```

Comment:
Good to add a couple sentence introduction here before jumping into specifics.
Imagine someone landing here via search and not necessarily knowing what a Jupyter notebook is or is used for or that there are VS Code for the Web or Codespaces options along with VS Code desktop

Reply:
Added intro!